coalib/output/ConsoleInteraction.py

```diff
@@ -548,20 +548,17 @@ def choose_action(console_printer, actions):
     :param actions:         Actions available to the user.
```

Comment:
commit message: better: "Shorten descriptions of actions" i think mentioning a file here would be redundant
regarding body: "The users do not read the long descriptions. It is, like this, easier for users to read the options and not ignore them."